package.build.ts

```diff
@@ -280,6 +280,21 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
                   + "non-directional, like Kakoune.",
                 ],
               },
+              activeModeDisplayTextTransform: {
```

Comment:
This defines `activeModeDisplayTextTransform` as a property set on modes, not a global configuration. You need to move this below and name it `"dance.activeModeDisplayTextTransform"` instead.